packages/drawer/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@littlespoon/drawer",
+  "version": "1.0.0",
+  "description": "Little Spoon drawer",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "build:flowtypes": "flowgen lib && rm -rf lib && mv exports lib && yarn build",
+    "clean": "rm -rf coverage exports lib",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "lint:tsc": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@littlespoon/button": "1.7.2",
+    "@littlespoon/icons": "1.8.0",
```

This should be updated to latest version:

```suggestion
    "@littlespoon/icons": "1.9.0",
```

---

Fixed in cf62f6651f04b78f882663b6401a8ca4df30fe29